README.md

```diff
@@ -53,6 +68,12 @@ Download [the latest JAR](https://github.com/ably/ably-java/releases) or grab vi
 compile 'io.ably:ably-java:0.8.4'
```

Comment:
Yes, but there are several references to 0.8.4 in README.md, and the "release notes" instruction says to update them when you are doing a new version. So it will get changed when someone does a release.